paasta_tools/chronos_tools.py

```diff
@@ -154,6 +154,10 @@ class UnknownChronosJobError(Exception):
     pass
 
 
+class MultipleChronosJobs(Exception):
```

Comment:
oh - are we even using this anymore? I can't find a place you're raising it. Should this be in some caller - ``chronos_serviceinit`` maybe?